src/exoscale/vinyl/store.clj

```diff
@@ -138,6 +162,29 @@
             (apply [_ context]
               (f (store-from-builder (::builder this) context)))))))
 
+(defn wrapped-runner
```

Comment:
A convenience wrapper around a `FDBDatabaseRunner` to ensure it behaves like a database.